readme.txt

```diff
@@ -35,6 +35,10 @@ We're always interested in your feedback and our [premium forums](https://theeve
 
 == Changelog ==
 
+= 1.2.1 2018-08-15 =
+
+* Fixed a bug due to which the post per page fix stopped working

```

Comment:
Please clarify the wording a bit, such as
> Fix - Posts per page customization works again
(Not the best but you get the idea - simplify)

Reply:
I tried your suggestions for the action hook / filter, but didn't work for me, so leaving it for you.